static/js/publisher/release.js

```diff
@@ -13,17 +14,31 @@ const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 const store = createStore(releases, composeEnhancers(applyMiddleware(thunk)));
 
 const initReleases = (id, snapName, releasesData, channelMapsList, options) => {
-  ReactDOM.render(
-    <Provider store={store}>
-      <ReleasesController
-        snapName={snapName}
-        channelMapsList={channelMapsList}
-        releasesData={releasesData}
-        options={options}
-      />
-    </Provider>,
-    document.querySelector(id)
-  );
+  if (!releasesData["error-list"]) {
+    ReactDOM.render(
+      <Provider store={store}>
+        <ReleasesController
+          snapName={snapName}
+          channelMapsList={channelMapsList}
+          releasesData={releasesData}
+          options={options}
+        />
+      </Provider>,
+      document.querySelector(id)
+    );
+  } else {
+    ReactDOM.render(
+      <div className="row">
```

Comment:
If there is an error in releases data maybe we can just render notification in HTML on server side, not try to run releases JS at all? We know that there is an error even in the python view, no need to pass it all the way to React.

Reply:
... :mindblownemoji: